std/bitmanip.d

```diff
@@ -37,18 +37,12 @@ version(unittest)
 }
 
 
-private string myToStringx(ulong n)
-{
-    enum s = "0123456789";
-    if (n < 10)
-        return s[cast(size_t)n..cast(size_t)n+1];
-    else
-        return myToStringx(n / 10) ~ myToStringx(n % 10);
-}
-
 private string myToString(ulong n)
 {
-    return myToStringx(n) ~ (n > uint.max ? "UL" : "U");
+    import core.internal.string;
+    UnsignedStringBuf buf;
+    auto s = unsignedToTempString(n, buf);
+    return cast(string)s ~ (n > uint.max ? "UL" : "U");
```

Comment:
It's most likely CTFE-only function, I'd suggest to keep things simple and just let it be for now.

Let's carefully investigate more impactful changes, thanks.